lib/components/output-area.js

```diff
@@ -17,8 +19,24 @@ const EmptyMessage = () => {
   );
 };
 
+const handleClick = () => {
+  //convert output to a plain js object
+  const output = toJS(kernel.outputStore.outputs[kernel.outputStore.index]);
```

Comment:
`kernel` isn't defined here so this fails tests. It should come from the store like below.

Reply:
:point_up: you can check if you caught the flow errors locally by running `npm run flow` from the root of your hydrogen project directory.

@rgbkrk I was thinking you could just move `handleClick` into the `outputArea` observer function, then add:

```js
  const handleClick = () => {
    if (!kernel || !kernel.outputStore) return;
```
to satisfy flow?